framework/include/mesh/FaceInfo.h

```diff
@@ -159,6 +159,18 @@ class FaceInfo
    */
   void computeCoefficients(const ElemInfo * const neighbor_info);
 
+  /**
+   * Computes the interpolation weights and similar quantities with the assumption
+   * that the face is on a boundary.
+   */
+  void computeCoefficients();
```

Comment:
can you call it computeBoundaryCoefficients ?
Otherwise it s easy to use the wrong one if you dont read the docstring

Reply:
Sure, good idea! Will rename it in the next commit!